reactioncommerce/api-migrations
# refactor: import migrations from npm package instead of internal files

Move migrations out of `/migrations/legacy-authorization`, and instead import them from the new `plugin-simple-authorization` npm package.

## Testing
- Run this branch of `reaction`, which uses `@reactioncommerce/plugin-simple-authorization`: https://github.com/reactioncommerce/reaction/pull/6096
- See migrations need to run
- Make sure `@reactioncommerce/plugin-simple-authorization` is installed in this project
- run `migrator migrate`
- see migrations work
